### DISCUSSION TITLE
Error: connect ECONNREFUSED

### DISCUSSION THREAD

**Discussion by Honzoraptor31415** [supabase/supabase#23337] [Questions] [answered]
My portfolio project on supabase just recentely stopped working for absolutely no reason. I always get this 

![dialog-screenshot](https://github.com/supabase/supabase/assets/151644171/c21c2021-5595-40f8-b607-0c946b7d5fe4)

error dialog in the supabase console and a `Failed to fetch` error in the browser console.

How do I fix this?
Or rather can I even fix it?
Can I at least get the table data from this project **somehow**?

**Comment by GaryAustin1** [comment, marked as answer]
You should certainly start a support request.
If a different browser also fails then not much you can do.

Note this may not impact the REST API or connecting to the database thru the database port.
